### PR TITLE
dash/docs-mathmodifier

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: docs/dashboards/data-mathmodifier.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+Data's MathModifier
```
Comment (goransle) [inline docs/dashboards/data-mathmodifier.md:1]: This Data?
image

Maybe just "MathModifer" or "Data module: MathModifier" would be clearer.

File: docs/dashboards/data-mathmodifier.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        'First Column': [ 1, 2, 3 ], // = Column A
+        'Second Column': [ 4, 5, 6 ], // = Column B
+        'Total Column': [ '=SUM(A1:B1)', '=SUM(A2:B2)', '=SUM(A3:B3)' ]
+    }
+}
+```
+
+
+
+Supported Operators & Function
```
Comment (goransle) [inline docs/dashboards/data-mathmodifier.md:82]: Function --> Functions. Maybe also Calculation --> Calculations in the above headers to be consistent?

File: docs/dashboards/data-mathmodifier.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
+Data's MathModifier
+===================
+
+The MathModifier provides different functions to run pre-defined formulas on a
+table, or to calculate the pre-filled formulas loaded into the table cells.
+MathModifier's integrated formula system supports a subset of known spreadsheet
+functions, but more can be defined by your needs.
+
+As in spreadsheets you can use cell and range references.  The MathModifier
```
Comment (sebastianbochan) [inline docs/dashboards/data-mathmodifier.md:9]: Double spaces

File: docs/dashboards/data-mathmodifier.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+* `A1`: One or more capital letters are the index for the column, while one or 
+  more digits are the index for the row.  The `$` syntax for fixed indexes is
+  also supported.
+
+* `A1:Z9`: Capital letters are the start index and end index for the columns of
+  the range, while digits are the start index and end index for the rows of the
+  range.
+
+* `R1C1`: __R__ followed by one or more digits is the index for the row, while
+  __C__ followed by one ore more digits is the index for the column.  This
```
Comment (sebastianbochan) [inline docs/dashboards/data-mathmodifier.md:21]: Double spaces. It appears in a few places, so please check it globally.

**Review by sebastianbochan — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]
LGTM

**Review by karolkolodziej — APPROVED** [review]

**Review by goransle — APPROVED** [review]

**Pull request merged**